systemtest/src/test/resources/connect-build/Dockerfile

```diff
@@ -0,0 +1,14 @@
+ARG KAFKA_VERSION
+ARG BASE_IMAGE=quay.io/strimzi/kafka:latest-kafka-${KAFKA_VERSION}
+
+FROM ${BASE_IMAGE}
+
+# to make KAFKA_VERSION arg available again after FROM, we need to specify it once more (different build stage)
+ARG KAFKA_VERSION
+
+USER root:root
+RUN mkdir -p /opt/kafka/plugins/file-sink
+
+ADD --chmod=444 https://repo1.maven.org/maven2/org/apache/kafka/connect-file/${KAFKA_VERSION}/connect-file-${KAFKA_VERSION}.jar /opt/kafka/plugins/file-sink/
```

Comment:
Should you copy it within the image instead of relying on an outside source? It helps to avoid outages, etc.

Reply:
You mean having the jars already in place in the `systemtest/src/test/resources` and copy it in the image, or move the download the download part to the script I created?

Reply:
The `connect-file-<version>.jar` file is IMHO already inside the image. Just not in the classpath. So you can IMHO just do `RUN cp /opt/kafka/libs/connect-file-3.6.0 ...` ... this is important since you use it from the image and do not need to care about the version, need to make the Maven repository configurable for RCs of Kafka etc.

Reply:
Ah okay, I will try it that way, thanks! :)